### ISSUE TITLE
bmp written is not supported by Chrome or Firefox

### ISSUE THREAD

**Issue by pammann-work** [SixLabors/ImageSharp#886]
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am running the latest version of ImageSharp
- [x] I have verified if the problem exist in both `DEBUG` and `RELEASE` mode
- [x] I have searched [open](https://github.com/SixLabors/ImageSharp/issues) and [closed](https://github.com/SixLabors/ImageSharp/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
When saving image as bmp it can not be seen with Chrome or Firefox, however it can be read by MS Paint.

This started to happen with 1.0.0-beta0006 when i compare the produced image with one from 1.0.0-beta0005 i can see that the bmp header is different....

[images.zip](https://github.com/SixLabors/ImageSharp/files/3085593/images.zip)

### Steps to Reproduce
``` c#
using (var image = Image.Load<Rgb24>(mem))
{
    // issue here
    image.SaveAsBmp(fullMem);
}
```

### System Configuration
Seen under Windows and Linux

- ImageSharp version: 1.0.0-beta0006
- Other ImageSharp packages and versions:
- Environment (Operating system, version and so on):
- .NET Framework version:
- Additional information:

**Comment by pammann-work** [comment]
![header_compare](https://user-images.githubusercontent.com/15227462/56222451-5af12b80-606c-11e9-8f2b-1c2d2b9cc126.png)

**Comment by JimBobSquarePants** [comment]
Odd. V4 bitmaps should be supported. @brianpopow do you have any insight here?

http://entropymine.com/jason/bmpsuite/bmpsuite/html/bmpsuite.html

**Comment by brianpopow** [comment]
we have indeed changed the bitmap header in beta0006 to V4 to support bitmaps with transparency. As @JimBobSquarePants said, chrome and firefox should support those.

I checked the imagesharp bitmap encoder and at first glance i can not spot an error. 
I was thinking maybe we should write V3 bitmap header as default and keep writing V4 headers as optional. I will look further into this the next days.

**Comment by JimBobSquarePants** [comment]
Thanks @brianpopow appreciate it! That sounds like a sensible plan.

**Comment by pammann-work** [comment]
update: gimp can also not read the image produced with beta006, it complains with:
"Unrecognized or invalid BMP compression format."

**Comment by pammann-work** [comment]
the problem is the incorrect 0x03, when i change it to 0x00 it works
![header_compare_2](https://user-images.githubusercontent.com/15227462/56350866-05339500-61cc-11e9-9d7d-8259d57b86c7.png)

**Comment by pammann-work** [comment]
issue comes most probably from BmpEncoderCore.cs / Line 131: "Compression = BmpCompression.BitFields".

Why is BitFields here used, the aren't the pixels anyway stored in memory byte by byte, thus "Compression = BmpCompression.RGB" should do the trick.

**Comment by brianpopow** [comment]
The v4 header does not seem to be the issue here. It seems to be combination of `V4 header + 24 bits per pixel + Bitfields compression` which chrome and firefox does not like. Bitfields with 32 bits per pixel does work with chrome, firefox and irfanview.

@pammann-work: as i said in my first post. The intention for using V4 header (with Bitfields) was to support transparency, but in your case with 24 bpp, there is no transparency anyway. Your hack for changing from bitfields (0x03) to no compression (0x00) works, because we use 8 bit per color channel with bitfields. This is essentially the same as uncompressed (without transparency). 
I wanted to change the bitmap header to default to V3, which implies no compression. V3 headers should be supported by almost all decoders.

**Comment by JimBobSquarePants** [comment]
Fixed with #889
